broker/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+kafka-python = "*"
+requests = "*"
+confluent-kafka = "*"
```

Comment:
Delete either Kafka Python or confluent

Reply:
Yeah, forgot to remove Kafka python because it didn't work